src/renderer/widgets/DelegateDetails/lib/utils.ts

```diff
@@ -26,3 +30,28 @@ export const getIdentityList = (identity: Identity) => {
     }
   }, []);
 };
+
+export const getDelegationsList = (delegations: Delegation[]) => {
+  const map = new Map();
```

Comment:
Looks dirty, do you really need Map here?

Reply:
Map helps reduce complexity here but I refactor it a bit